src/material-experimental/mdc-theming/_all-theme.scss

```diff
@@ -25,6 +26,7 @@
     @include mat-mdc-list-theme($theme-or-color-config);
     @include mat-mdc-menu-theme($theme-or-color-config);
     @include mat-mdc-progress-bar-theme($theme-or-color-config);
+    @include mat-mdc-progress-spinner-theme($theme-or-color-config);
```

Comment:
Can you remove the changes to `_all-theme.scss` in this PR? This makes the PR not merge-safe since some people are using mdc theming. They can go back in a follow-up PR.